vividus-plugin-json/src/main/java/org/vividus/json/steps/JsonSteps.java

```diff
@@ -363,16 +368,24 @@ private void executeStepsForFoundJsonElements(String json, String jsonPath, Comp
     public void assertValueByJsonPath(String json, String jsonPath, String comparisonRule, Object expectedData)
     {
         getDataByJsonPathSafely(json, jsonPath, true).ifPresent(jsonByPath -> {
+            List<String> equalAndNotEqualComparisonRules = Arrays.asList("IS_EQUAL_TO", "IS_NOT_EQUAL_TO");
             if (jsonByPath.isEmpty())
             {
-                validateEqualsComparison(comparisonRule, null, expectedData);
-                assertThat(jsonPath, null, equalTo(expectedData));
+                validateComparisonRuleAndNullValue(jsonPath, comparisonRule, equalAndNotEqualComparisonRules,
+                        null, expectedData);
```

Comment:
why is it changed?

Reply:
For possibility to execute step using "is not equal to":
```
Given I initialize scenario variable `json` with value `
{
    "persons": {
        "nemo": null
    }
}
`
Then JSON element value from `${json}` by JSON path `$.persons.nemo` is not equal to `2`
```

Reply:
are you sure these lines of code handle this case?

Reply:
yes, I'm sure. Now this line:
`validateComparisonRuleAndNullValue(jsonPath, comparisonRule, null, expectedData);`